UI/src/view/dynamicAssetForm.js

```diff
@@ -906,10 +906,38 @@
             return '<span class="marker">' + sideCode + '</span>';
         }
 
+        var userInformationLog = function() {
+            var selectedAsset = _assetTypeConfiguration.selectedLinearAsset;
+            var authorizationPolicy = _assetTypeConfiguration.authorizationPolicy;
 
-      var informationLog = function (date, username) {
-        return date ? (date + ' / ' + username) : '-';
-      };
+            var hasMunicipality = function(linearAsset) {
+                return _.some(linearAsset.get(), function(asset){
+                    return authorizationPolicy.hasRightsInMunicipality(asset.municipalityCode);
+                });
+            };
+
+            if(authorizationPolicy.isMunicipalityMaintainer() || authorizationPolicy.isElyMaintainer()) {
+                if(!hasMunicipality(selectedAsset)) {
+                    return '' +
+                      '<div class="form-group user-information">' +
+                      '<p class="form-control-static user-log-info"> Käyttöoikeudet eivät riitä kohteen muokkaamiseen. Voit muokata kohteita vain omalla toimialueellasi. </p>' +
+                      '</div>';
+                } else {
+                    return '';
+                }
+            } else if(checkAuthorizationPolicy(selectedAsset)) {
+                return '' +
+                  '<div class="form-group user-information">' +
+                    '<p class="form-control-static user-log-info"> Käyttöoikeudet eivät riitä kohteen muokkaamiseen </p>' +
+                  '</div>';
+            } else {
+                return '';
+            }
+        };
+
+        var informationLog = function (date, username) {
+           return date ? (date + ' / ' + username) : '-';
+        };
```

Comment:
the information log is missing for link property form, manoeuvre form, mass transit stop form, speed limit form